charts/client/chart.scss

```diff
@@ -249,6 +254,10 @@ $zindex-IndicatorDropdown: 50;
             border-left: 1px solid #e1e1e1;
             display: table-cell;
             height: $footerRowHeight;
+
+            &.download {
```

Comment:
nit: what's the rationale behind 30%? I'm not quite sure. also, maybe name this downloadButton or something a little more descriptive?

Reply:
30% looked nice and dynamically resized better than an explicit pixel count.
Will rename to `download-tab-button`!